### PR TITLE
fix minor validation bug in compatability

### PR DESCRIPTION
**Pull request by grimreaper**
don't think this is anything other than a newly working validation, but maybe I missed where else this would get triggered.

### CODE REVIEW

File: code/src/java/pcgen/io/Compatibility.java [hunk 1]
```diff
@@ -112,15 +122,15 @@ else if (!max.equals(min))
 			return null;
 		}
 		int closeParenLoc = first.lastIndexOf(')');
-		if (openParenLoc == -1)
+		if (closeParenLoc == -1)
```
Comment (grimreaper) [inline code/src/java/pcgen/io/Compatibility.java]: this is the actual bug

Comment (thpr) [inline code/src/java/pcgen/io/Compatibility.java]: Interesting that was missed originally.  This was originally coded for the 5.12->5.14 conversion, so probably not really used anymore

Comment (grimreaper) [inline code/src/java/pcgen/io/Compatibility.java]: should it just be removed? I'm not sure how much of this is still useful

File: code/src/java/pcgen/io/Compatibility.java [hunk 2]
```diff
@@ -132,8 +142,7 @@ else if (!max.equals(min))
 			return null;
 		}
 		context.commit();
-		PersistentTransitionChoice<?> ptc = applied.getListFor(ListKey.ADD).get(0);
-		return ptc;
+		return applied.getListFor(ListKey.ADD).get(0);
```
Comment (thpr) [inline code/src/java/pcgen/io/Compatibility.java]: I'm concerned this won't compile.  There have been similar changes made in other places that when inlined cause various compilers to complain... At what level has this been tested?

File: code/src/java/pcgen/io/Compatibility.java (hunk 1 above)
Comment (thpr) [inline code/src/java/pcgen/io/Compatibility.java]: It's still needed

**Pull request merged**